docs/webaverse/.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do we need visual studio stuff here?

Reply:
don't think so, wonder if I could just delete the .vs folder